src/common/motor.rs

```diff
@@ -302,7 +302,7 @@ impl Status for FakeMotor {
     }
 }
 
-impl Stoppable for FakeMotor {
+impl Actuator for FakeMotor {
```

Comment:
I was surprised to not see an `is_moving` for `FakeMotor`

Reply:
I actually just forgot, maybe this is a sign that I shouldn't include default implementations in the `Actuator` trait?

Reply:
I think it's better not to have default for these because
1) The compiler will catch it if not implemented
2) We won't have to jump around between `common` and the specific implementation to see what the current implementation is